game_logic.cpp

```diff
@@ -17,22 +17,52 @@ Board createBoard() {
 		Player player;
 		player.mineCount = mineCount;
 		player.id = p + 1;
-		if (board.gameType == PVE) {
+		if (board.gameType == PVE) { // if game is PVE and the player is not the first one, they're a bot (TODO: make the user able to choose which players will be human and which will be bot)
 			if (p > 0) {
 				player.isAI = true;
 			}
+		} else if (board.gameType == EVE) { // in EVE, all players are bots
+			player.isAI = true;
 		}
 		board.players.push_back(player);
 	}
+	if (board.gameType == EVE) {
+		setAwaitUserInput(false);
+	}
+	return board;
+}
+
+Board createBoard(int gameTypeValue, int width, int height, int mineCount, int playerCount) { // for debugging purpouses

```

Comment:
#should make this the only overload and supply default values for all parameters to avoid duplicating the code